src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTests.cs

```diff
@@ -117,7 +107,8 @@ public static void MessageBody_Elements_CustomerElement_Value_Matches()
                     }
                     else
                     {
-                        errorBuilder.AppendLine(String.Format("Comparison Failed. Expected: {0}, Actual: {1}", MessageContractConstants.customerElementValue, reader.Value));
+                        Assert.True(false, 
+                            string.Format("Comparison Failed. Expected: {0}, Actual: {1}", MessageContractConstants.customerElementValue, reader.Value));
```

Comment:
This should be replaceable with Assert.Equal, as that will also output the expected and actual value - it does away with the string.Format entirely